scripts/create-sentry-release

```diff
@@ -16,21 +19,20 @@ if [ -z "${SENTRY_AUTH_TOKEN:-}" ]; then
   echo 'No Sentry auth token found' && exit 1
 fi
 
-echo 'Creating a new release...'
-curl \
-  "https://sentry.io/api/0/organizations/sentry/releases/" \
-	-X POST \
-  -H "Authorization: Bearer ${SENTRY_AUTH_TOKEN}" \
-	-H "Content-Type: application/json" \
-	--data "
-    {
-      \"version\":\"${VERSION}\",
-      \"refs\": [{
-        \"repository\": \"${GITHUB_PROJECT}\",
-        \"commit\": \"${VERSION}\"
-      }],
-      \"projects\": [\"${SENTRY_PROJECT}\"]
-    }"
-
-echo
-echo 'Release created.'
+echo 'Installing sentry-cli...'
+mkdir $BIN_DIR
+curl -sS https://sentry.io/get-cli/ | INSTALL_DIR=$BIN_DIR bash
+export PATH=$PATH:$BIN_DIR
+sentry-cli --version
+
+echo 'Pulling debug information from semaphore image...'
+docker pull "${DOCKER_IMAGE}"
+docker run --rm --entrypoint cat "${DOCKER_IMAGE}" /opt/semaphore-debug.zip > semaphore-debug.zip
+
+echo 'Uploading debug information...'
+sentry-cli upload-dsym ./semaphore-debug.zip
```

Comment:
@tonyo Do **not** use `upload-dsym`, this command is deprecated and has slighly different behavior than `upload-dif`.